### ISSUE TITLE
Muted group still have red unread messages count on app icon

### ISSUE THREAD

**Issue by squadgazzz** [peter-iakovlev/Telegram#244]
Hello! Only one group for some reason still have red unread messages count on app icon. iOS.
How can I avoid it?

**Comment by ghost** [comment]
Hi. 

Even if bot is muted, it’s messages will be displayed in red badge. Isn’t it your situation?

If not, maybe leaving and joining back or re-muting can help.

**Comment by alitk** [comment]
I think this problem is for all groups (not supergroups) and only in iOS.

**Comment by squadgazzz** [comment]
No. Only single group still have red notification on application's icon.
Every other mutted groups work fine.
Telegram X don't have this problem at all.
I have already tried to rejoin, remute this chat and that didn't help at
all.

*--*
Sincerely,
Ilya Zhavoronkov

**Comment by alitk** [comment]
Yes Telegram X does not have this problem. The reason is that this is the only group that you have which is not a supergroup.

**Comment by squadgazzz** [comment]
Thanks everyone. The problem resolved by making the group super. I hope this will be fixed in future.

Отправлено с iPhone

**Comment by hajk1** [comment]
I have the same issue which occurs only in a specific group just in iOS. Android and Telegram desktop does not have this issue.
The only strange thing about this group is that it has no admin* user

**Comment by MaryamMahmoodi** [comment]
I have the same issue in  iOS and Telegram desktop on Linux(Ubuntu 18) .
I confirm this sentence: ("The only strange thing about this group is that it has no admin* user")
Is there no solution?

**Comment by bcwingnut** [comment]
Just close **Include Muted Chats**
